gradle/libs.versions.toml

```diff
@@ -59,7 +59,7 @@ mongo = "4.11.5"
 netty = "4.2.7.Final"
 newrelic-api = "5.14.0"
 # Kotlin 1.7 sample will fail from OkHttp 4.12.0 due to okio dependency being a Kotlin 1.9 module
-okhttp = "4.11.0"
+okhttp = "4.12.0"
```

Comment:
I guess the comment above here is stale? Because we're using a newer Kotlin plugin version now? I don't know.